src/main/java/com/dougnoel/sentinel/pages/PageManager.java

```diff
@@ -3,16 +3,17 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
-import org.openqa.selenium.NoSuchFrameException;
-import org.openqa.selenium.NoSuchWindowException;
-import org.openqa.selenium.NotFoundException;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.openqa.selenium.By;
 import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchFrameException;
+import org.openqa.selenium.NoSuchWindowException;
+import org.openqa.selenium.NotFoundException;
```

Comment:
Let's just revert this file to what it was before the changes were made and leave it untouched.

Reply:
Done.